pnpm-lock.yaml

```diff
@@ -1,4 +1,4 @@
-lockfileVersion: '6.1'
+lockfileVersion: '6.0'
```

Comment:
@airhorns I tried downgrapding `pnpm` but this kept happening. Which `pnpm` version are you on?

Reply:
I'm on `8.15.4`.

Reply:
Weird. I updated `main` to lock us all to use the same version of pnpm via corepack. If you rebase on top of `main` and then re-run pnpm, I'll take whatever lockfile that produces!